bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/SetupListener.java

```diff
@@ -56,7 +56,7 @@ private boolean compatible(Alloc pred, Alloc succ) {
 		return 
 			pred != null &&
 			pred.getObs().getInstrumentString().equals(succ.getObs().getInstrumentString()) &&
-			pred.getObs().getCoords().equals(succ.getObs().getCoords());
+			pred.getObs().getCoords(pred.getEnd()).equals(succ.getObs().getCoords((succ.getEnd())));
```

Comment:
Ok, I think now I'm getting what you mean. This will do, right? `pred.getObs().getCoords(pred.getEnd()).equals(succ.getObs().getCoords((pred.getEnd())));`